docs/how-to-contribute.md

```diff
@@ -76,6 +76,14 @@ kubectl -n $KUBE_NAMESPACE  apply -f examples/datadogagent/datadog-agent-minimum
 
 The Operator should start deploying the `agent` and `cluster-agent`.
 
+## Golang version update
+
+The Go version is defined in several files. To ensure all relevant files are updated, it is recommended to use the script `make update-golang`:
+
+1. Update the Go version in the `go.work` file.
+2. Run the command `make update-golang`, which patches all files that contain the Go version.
+
+If the golang version is used in a new file (for example in a new `Dockefile`) the script `hack/update-golang.sh` needs to be updated to handle this new file in the golang version update process.
```

Comment:
```suggestion
If the Golang version is used in a new file (for example in a new `Dockerfile`) the script `hack/update-golang.sh` needs to be updated to handle this new file in the Golang version update process.
```

Reply:
This is cherry-picked and not going in main so so will not add it here.